test/test_ops_gradients.py

```diff
@@ -44,7 +45,9 @@ def is_inplace(variant):
             return variant is op.get_inplace()
 
         include_conjugated_inputs = op.test_conjugated_samples and dtype.is_complex
-        samples = op.sample_inputs(device, dtype, requires_grad=True, include_conjugated_inputs=include_conjugated_inputs)
+
+        samples = op.sample_inputs(device, dtype, requires_grad=True, include_conjugated_inputs=include_conjugated_inputs,
+                                   small_inputs_only=os.environ.get('PYTORCH_TEST_WITH_SLOW_GRADCHECK', "0") == "1")
```

Comment:
nit: have a small utility function in testing _internal to avoid copy-pasted env variable name!